.circleci/config.yml

```diff
@@ -31,15 +30,15 @@ jobs:
       - save_cache:
           paths:
             - ./venv
-            - /opt/circleci/.pyenv
-          key: v1-dependencies-3.6.3-{{ checksum "setup.py" }}-{{ checksum "dev-requirements.txt" }}
-        
+          key: v1-dependencies-3.7.0-{{ checksum "setup.py" }}-{{ checksum "dev-requirements.txt" }}
+
       - run:
           name: run tests
           command: |
             source venv/bin/activate
             mkdir -p test-reports/unit
-            py.test --junitxml=test-reports/unit/results.xml tests/
+            py.test --cov=hubploy --junitxml=test-reports/unit/results.xml tests/
+            codecov
```

Comment:
I'm not sure, but will this command fail if codecov succeeds after a failing py.test for example? In the chartpress repo that caused builds to not fail as they should before because a `git --no-pager diff` command always succeeded as a last command in a multiline string, even though previous commands in the multiline string failed.

https://github.com/jupyterhub/chartpress/commit/97ebe4bd4b6be3101088d236feee0bcd7ba9fc6a

Reply:
Ah nice! I'm not sure, but I split it up to a new step anyway :)